content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -41,7 +41,7 @@ This page lists some of the available add-ons and links to their respective inst
 ## Visualization &amp; Control
 
 * [Dashboard](https://github.com/kubernetes/dashboard#kubernetes-dashboard) is a dashboard web interface for Kubernetes.
-* [Weave Scope](https://www.weave.works/documentation/scope-latest-installing/#k8s) is a tool for graphically visualizing your containers, pods, services etc. Use it in conjunction with a [Weave Cloud account](https://cloud.weave.works/) or host the UI yourself.
+* [Weave Scope](https://www.weave.works/docs/scope/latest/) is a tool for visualizing your containers, Pods, Services and more.
```

Comment:
As this article intend to install plugins i think this would be a better reference to it https://www.weave.works/docs/scope/latest/installing/

Reply:
Hello @reylejano @divya-mohan0209 I believe the [link](https://www.weave.works/docs/scope/latest/installing/) section is mentioned on this [page](https://www.weave.works/docs/scope/latest/), Please suggest what changes need to be made.

Reply:
Hello @kadtendulkar .
The existing link (`https://www.weave.works/documentation/scope-latest-installing/#k8s`) redirects to `https://www.weave.works/docs/scope/latest/installing/#k8s`, similar to the link suggested by @Kartik494 .

Do you want to use this link (`https://www.weave.works/docs/scope/latest/installing/#k8s`)?